### DISCUSSION TITLE
Rollladen Szene anfahren

### DISCUSSION THREAD

**Discussion by gt-four** [brutella/hkknx-public#250] [Q&A]
Hallo zusammen,

ich kann unsere Rollläden (mit KNX SMI Aktoren) mit den üblichen Befehlen Auf, Stopp, Ab und einer beliebigen Behangposition in % noch zusätzlich 2 Szenen mit fest programmierten Behangpositionen anfahren.

In den Schlafräumen fahren wir abends die Rollläden mit einer der Szenen auf 80 %. Sie sind dann zu, aber nicht ganz geschlossen (Lüftungsmöglichkeit).

Ich kann dies mit den Tastsensoren in den Zimmern problemlos lösen. Hier finde ich keine Möglichkeit, den Rollladen in eine fest programmierte Position zu fahren. Sie fahren immer komplett zu. In der Konfiguration "Rollladen" sehe ich gar keine Möglichkeit, als "Garagentor" dachte ich geht es, da einige ihre Tore zum Lüften der Garage auch einen Spalt auffahren. Gerade hier lohnt sich eine fest gespeicherte Position. Aber auch hier finde ich keine Möglichkeit, denn ich kann zwar getrennte Gruppenadressen für Öffnen und Schließen eintragen, aber bei beiden nur den Objektwerkt 1 senden. Für das Hochfahren benötige ich den Objektwert 0.

Habe ich eine Möglichkeit übersehen?

**Comment by chef76** [comment]
Ja, da hast Du mehrere Möglichkeiten übersehen.
1. Du kannst in Homekit eine Szene anlegen, die die Jalousie auf 80% fährt
2. Du legst in der Brücke einen Schalter vom Typ Szene an, der genau das macht, was Dein Physikalischer Tstsensor macht. Wenn es eine echte KNX-Szene Szene ist, musst Du als Datentyp die Szenen-ID nehmen oder wenn Du nur eine 0 oder 1 brauchst, musst du ganz nach unten scrollen: 
![image](https://user-images.githubusercontent.com/69462419/211985975-2819342d-96a1-4d9e-9090-2e6da9ddcd4f.png)

**Comment by gt-four** [comment]
Hi,

danke für die Infos und deine Hilfe. Habe heute beides ausprobiert und es funktioniert auch beides.

Bei der Anlage der Szene in HomeKit war es etwas schwierig zu verstehen, dass man die Jalousie erst auf 80% fahren muss und diese Position mit der Szene dann so speichert. Habe immer eine Möglichkeit gesucht, da irgendwo einen Wert einzugeben…

Die Möglichkeit eines Tasters, der nur eine 0 oder 1 sendet, hatte ich in der Brücke tatsächlich übersehen. Funktioniert auch! Echte KNX Szenen habe ich bisher noch nie in der ETS angelegt, sondern fahre die betreffenden Jalousien über eine einzige Gruppenadresse in die in den Rollladenaktoren gespeicherte Position.

Die grundsätzliche Frage wird sein, ob man Szenen in der ETS anlegt oder alle Jalousien einzeln in der Brücke hat und in HomeKit entsprechende Szenen anlegt. Wenn alle Geräte in HomeKit abgebildet werden, kann das ganz schnell unübersichtlich werden.

Gruß
Christian
Am 12. Jan. 2023, 06:36 +0100 schrieb chef76 ***@***.***>:

**Reply by chef76** [reply]
Schön, dass es klappt!
Es ist Geschmackssache, ob du es so oder so machst. Eine HomeKit Szene hast du halt schneller angepasst. Das einzige Problem ist eine saubere Namenskonvention, da die Liste schnell lang wird. 

Gruß 
Christian 😜
